lib/Core/AnalyticEvents/analyticEvents.ts

```diff
@@ -51,7 +51,8 @@ export enum DataSourceAction {
   removeFromPreviewButton = "Remove from preview button",
   removeFromWorkbench = "Remove from workbench",
   removeAllFromWorkbench = "Remove all from workbench",
-  addFromDragAndDrop = "Add from Drag and Drop"
+  addFromDragAndDrop = "Add from Drag and Drop",
+  addFromShareUrl = "Add from share URL"
```

Comment:
This event is fired when a dataset is added from share URL, but also when it's added from an initialisation file. Maybe this action should be "Add from share or init source"